apps/daimo-mobile/src/i18n/languages/en.ts

```diff
@@ -69,12 +71,32 @@ export const en = {
         thirdPara: () =>
           `Transactions cost a few cents. Daimo sponsored this transfer, making it free.`,
       },
+      landlineDeposit: {
+        firstPara: () =>
+          "This transaction transfers funds from your connected bank account to your Daimo account.",
+        secondPara: () =>
+          "Once the funds are received by our partner, we will make an on-chain transfer to deposit the funds to your Daimo account.",
+        thirdPara: () =>
+          "Bank transfers normally cost a few dollars. Daimo sponsored this transfer, making it free.",
```

Comment:
🔥 

this is all written future tense. should it change once the transfer lands?